tests/README.md

```diff
@@ -0,0 +1,16 @@
+# Install helm unittest
+Install and get [helm unittest](https://github.com/lrills/helm-unittest) running. 
```

Comment:
I haven't dug into these tests in this review but I noticed that this tool may not be maintained...
https://github.com/lrills/helm-unittest/issues/102

any concerns about this? I'm fine merging these in right now, it would be a shame if they had to be deleted in the future though due to `unittest` falling out of favour with the community. wdyt?

Reply:
Doesn't work with helm 3 either: https://github.com/lrills/helm-unittest/pull/104

Reply:
I removed the test, you are right the tool seems to be unmaintained. I want to avoid problems in the future.